server/src/modules/contests/contests.controller.ts

```diff
@@ -27,10 +27,10 @@ export class ContestsController {
 
   // GET /competitions?region=...
   @Get()
-  async getContests(@Query('region') region: string) {
+  async getContests(@Query('region') region: string, @Query('eventId') eventId: string) {
```

Comment:
Can you make region and eventId optional here, and also update the comment on line 28 to this?
`// GET /competitions(?region=...&eventId=...)`